server/src/controllers/seasonController.ts

```diff
@@ -145,6 +138,9 @@ seasonController.get('/', async (req: Request, res: Response) => {
   }
 });
 
+seasonController.use(checkJwt);
+seasonController.use(checkScopes);
```

Comment:
I imagine that these should be above the `post` request

Reply:
I  created a separate ticket for that I think the controller itself needs to be rearranged, I've also noticed a couple other random routes that need to be moved so I thought I'd do them all together.

Reply:
Ok, that sounds good 👍🏻